project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.7.0")
 addSbtPlugin("org.tpolecat" % "tut-plugin" % "0.4.8")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "latest.release")
```

Comment:
I think you still need to point each library to their public scaladoc URL. In some projects I did something like this in order to make my scaladoc point to akka docs:

```
autoAPIMappings := true
apiMappings := findManagedDependency("com.typesafe.akka", "akka-actor").value.map {
  case (module, jar) => jar -> url(s"http://doc.akka.io/api/akka/${module.revision}/")
}.toMap
```

The plugin added by @melrief seems to do just that for a handful of common libraries.